python/kwiver/vital/util/pybind11.h

```diff
@@ -37,84 +37,99 @@
 #include <pybind11/pybind11.h>
 
 namespace kwiver {
+
 namespace vital {
+
 namespace python {
 
 /*
  * NOTE: These classes and macros have just been lifted from pybind11 and
  * rewritten to use the Python interpreter's standard PyGILState_* API, because
  * of the inherent compatibility issues in the pybind11 version. In the future,
- * we would like to get similar classes and macros merged into upstream pybind11
+ * we would like to get similar classes and macros merged into upstream
+ *pybind11
  * so that they become standard.
  */
 
 class gil_scoped_acquire
 {
-  public:
-    gil_scoped_acquire()
-    {
-      state = PyGILState_Ensure();
-    }
-
-    ~gil_scoped_acquire()
-    {
-      PyGILState_Release( state );
-    }
-
-  private:
-    PyGILState_STATE state;
+public:
+
```

Comment:
Sigh, bug... I'll leave it up to you if you want to fix this even though Uncrustify will try to break it again (several spots in this file) or leave it alone for now. At some point when this bug gets fixed, I expect Uncrustify will remove these.

Reply:
In my opinion, perhaps its best to keep it this way again until the bug gets fixed so developers dont rely on it being there.